app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt

```diff
@@ -100,11 +102,10 @@ fun LoginPage(
                     }
                 }
                 when (vm.wordpressState) {
-                    // A comprendre pourquoi le apploader ne s'affiche pas
                     is WordpressUiState.LOADING -> AlertDialog(
                         onDismissRequest = { /*TODO*/ }
                     ) {
-                        AppLoader()
+                        //AppLoader()
```

Comment:
If it's not trigger, we can delete this part of code

Reply:
Il keep it because of the TODO, maybe you wanted to add something here ?

Reply:
No, the comment is to find why is doesn't work but you have develop another way so it's good for me